test/tables.jl

```diff
@@ -0,0 +1,42 @@
+module TestTables
```

Comment:
Could you please add tests for the following things (also in combination):

* columns containing `missing` like `[1, missing]`;
* columns having a mixed type, e.g. `Union{String, Char, Missing}["A", 'a', missing]`, `Union{String, Char, Missing}["A", "B", "C"]`

I would think it would be good to test against three things:

* if we have proper column names (this is done I think);
* if we have proper values;
* if we have proper column types after conversion and back.

Also it would be good to test conversions to `rowtable` and `columntable` in both ways.